docs/user/implement_env.rst

```diff
@@ -1,50 +1,68 @@
 .. _implement_mdp:
 
-===========================
-Adding a New Environment
-===========================
+================================
+Working with Garage Environments
```

Comment:
Please keep the title 'Add a New Environment' because this is a tutorial

If you want to write the reference guide for environments, that's a different article.